json-schema/schema.json

```diff
@@ -140,6 +141,27 @@
                 }
               }
             }
+          },
+          {
+            "$comment": "Defines PMTiles links",
+            "if": {
+              "properties": {
+                "rel": {
+                  "const": "pmtiles"
+                }
+              }
+            },
+            "then": {
+              "properties": {
+                "pmtiles:layers": {
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "minLength": 1
```

Comment:
should this be 0 in the case it's a raster tileset?

Reply:
No, in this case you just omit the array or provide an empty array, but not an array with an empty string.

Reply:
Looks like it, but I want to see the person who complains about unsupported empty layer names ;-)